frontend/src/editor/alert/alert.component.ts

```diff
@@ -14,11 +14,11 @@ export class AlertComponent implements OnInit, OnDestroy {
     private operations: OperationsService
   ) { }
 
+  private alertStatus: Subscription;
   alert: AlertModel;
-  alertStatus: Subscription;
 
   ngOnInit() {
-    this.alertStatus = this.operations.sendStatus.subscribe(res => {
+    this.alertStatus = this.operations.observeStatus$.subscribe(res => {
```

Comment:
potrzebne jest przypisane do zmiennej? Skoro w sub res przypisujesz do this.alert ?

Reply:
tak, ponieważ odsubskrybowuje na ondestroyu